site/build.gradle

```diff
@@ -126,16 +126,41 @@ class ApiIndexTask extends DefaultTask {
             def href = "https://javadoc.io/doc/com.linecorp.armeria/armeria-javadoc/latest/${relativePath}"
 
             def result = []
-            def name = fqcn
-            for (;;) {
-                result.add([name, href])
-                def dotIndex = name.indexOf('.')
-                if (dotIndex < 0) {
-                    break
+            result.add([fqcn, href])
+
+            def simpleClassName = fqcn.substring(fqcn.lastIndexOf('.') + 1)
+            result.add([simpleClassName, href])
+
+            def br = new BufferedReader(new FileReader(path.toFile()))
+            def line = br.readLine()
+            // Read a file to retrieve the links of methods.
+            while (line != null) {
+                if (line.startsWith('<h3><a id="')) {
+                    // The length of `<h3><a id="` is 10
+                    def methodSignatureEndingIndex = line.indexOf('"', 11)
```

Comment:
How about using regular expression?

Reply:
Or How about using Jsoup? https://jsoup.org/cookbook/input/load-document-from-file
We could method name using CSS Selector easily. It is fast to parse local HTML(tested it locally).

```groovy
// in buildscript.dependencies
classpath 'org.jsoup:jsoup:1.13.1'

Document doc = Jsoup.parse(path.toFile(), "UTF-8")
doc.select("h3 > a").each {
    println it.text()  // get method names
}
```

Reply:
I think that will include other lines such as
```
<h3>Methods inherited from class&nbsp;java.lang.<a href= ...
```
So I guess I have to stick with `<h3><a id="` this expression.
Let me use regex. 😉

Reply:
Just out of curiosity, `.detail > h3 > a:not([id^=<init>])` works. 🤣

Reply:
Oh didn't know I can do it in that way. Let me try. Thanks!